pkg/kubelet/apis/kubeletconfig/v1beta1/types.go

```diff
@@ -120,10 +154,14 @@ type KubeletConfiguration struct {
 	// the certificates.k8s.io API. This requires an approver to approve the
 	// certificate signing requests. The RotateKubeletServerCertificate feature
 	// must be enabled.
+	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
+	// it may disrupt components that interact with the Kubelet server.
```

Comment:
may disrupt the automatic renewal of kubelet server certificates is more accurate I guess. In the long term, I that means as you say here it will eventually disrupt components talking to the kubelet server (like the API server). What default timeframe kubelet server certs are valid for I don't know.

Reply:
Thanks, updated the comment to be more specific.